queries.sql

```diff
@@ -40,4 +40,12 @@ SELECT count(*) FROM animals WHERE escape_attempts > 0;
 SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) from animals GROUP BY species;
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-30' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-30' GROUP BY species;
+
+SELECT animals.name, owner_id, full_name FROM animals JOIN owners ON owners.id = animals.owner_id WHERE owner_id = 4;
+SELECT animals.name, species_id, species.name FROM animals JOIN species ON species.id = animals.species_id WHERE species_id = 1;
+SELECT animals.name, owner_id, full_name FROM animals JOIN owners ON owners.id = animals.owner_id;
+SELECT animals.name,species_id,species.name FROM animals JOIN species ON species.id = animals.species_id;
```

Comment:
- [ ] The question here is "How many animals are there per species?" the expected answer is the species names (pokemon and digimon) and the `count` of animals in each species 💯 Kindly adjust your query and the screenshot to answer the question correctly 😄